suture.go

```diff
@@ -273,14 +248,12 @@ func New(name string, spec Spec) (s *Supervisor) {
 	return
 }
 
-func serviceName(service Service) (serviceName string) {
-	stringer, canStringer := service.(fmt.Stringer)
-	if canStringer {
-		serviceName = stringer.String()
-	} else {
-		serviceName = fmt.Sprintf("%#v", service)
+func serviceName(service Service) string {
```

Comment:
I believe not using named returns and using `if (...) { return ... }; return ...` is slightly more idiomatic.